bin/build_container_image

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+REGISTRY=${REGISTRY:-"docker.io"}
+ORGANIZATION=${ORGANIZATION:-"manageiq"}
+IMAGE_NAME=$REGISTRY/$ORGANIZATION/httpd_configmap_generator
+
+set -e
+
+docker build . -t localhost/build:latest
+
+VERSION=$(docker run --rm --entrypoint bash localhost/build:latest -c "ruby -e \"require 'httpd_configmap_generator'; puts HttpdConfigmapGenerator::VERSION\"")
+
+set -v
+docker tag localhost/build:latest $IMAGE_NAME:$VERSION
```

Comment:
This one is a little different.

The docker build always pulls in the latest released gem as opposed to source. So, on a PR to master, this will overwrite the existing release tag. I think master should only write latest, unless this is a tag build. Not sure how to differentiate, but maybe there's some TRAVIS_ env var.

Reply:
Also the tags on docker hub for this one have the leading `v` (e.g. `v0.3.2`)